### DISCUSSION TITLE
Code completion and IntelliSense

### DISCUSSION THREAD

**Discussion by TheDrone7** [SeaQL/sea-orm#786] [Q&A]
While using sea orm within CLion, I don't get any code completions for sea orm entities.

Is this a known issue or does it have a workaround solution?

**Comment by billy1624** [comment]
Hey @TheDrone7, I'm not sure what kind of code completion do you mean. Personally, I use VScode with Rust Analyser for day to day development and it works well.

Is this related?
- https://github.com/intellij-rust/intellij-rust/issues/8610

**Reply by TheDrone7** [reply]
It's similar but the thing is I don't see any errors. I just don't get code suggestions.
